Makefile

```diff
@@ -1,4 +1,12 @@
+keys:
+	mkdir keys
+	openssl genrsa -out keys/private.key 2048
+	openssl rsa -in keys/private.key -pubout -out keys/public.key
+	chmod 660 keys/private.key keys/public.key
 server:
 	docker run -itd --rm -v $$(pwd):/app -w /app/public -p 8080:8080 php:7.2-alpine -S 0.0.0.0:8080
 unittest:
-	docker run -it --rm -v $$(pwd):/app -w /app php:7.2-alpine vendor/bin/phpunit
\ No newline at end of file
+	docker run -it --rm -v $$(pwd):/app -w /app php:7.2-alpine vendor/bin/phpunit
+
+.SILENT: keys server unittest
```

Comment:
Why the silent flags?

Reply:
Could do without, personally I don't care about seeing the commands.

Reply:
I agree regarding the key commands, I'd like to see the output of my PHPUnit commands for example :).